home.md

```diff
@@ -3,3 +3,10 @@
 Virtual Science Forum is a volunteer-run open platform aiming to accommodate organization of online conferences.
 We provide support and hosting of virtual conference sessions.
 Additionally we develop best practices and provide detailed guidelines for all community members.
+
+## Where we stand
```

Comment:
I like the inclusion of this section but not sure about the title, unfortunately I don't have a better one myself. I drop here some ideas:

- Prospects of VSF
- Expanding VSF

Any ideas @everthemore , @greplova , @vfatemi ?